api/build/build.yaml

```diff
@@ -2694,6 +2694,14 @@ components:
           type: string
         energy_type:
           type: string
+          enum:
+            - Electric
```

Comment:
These all need to follow ENUM format of UPPER_CASE

Reply:
enums updated to upper case